src/main.cpp

```diff
@@ -385,7 +458,7 @@ std::string parse_options(int argc, char *argv[], Options *options) {
 }
 
 void usage() {
-	std::cerr << "usage: pdf2msgpack [--pages=a-b] <filename>" << std::endl;
+	std::cerr << "usage: pdf2msgpack [--meta-only] [--pages=a-b] <filename>" << std::endl;
```

Comment:
Thanks. `[--meta-only]` wins, yes, but not because it's last but because its `-only`.

Will add `[--bitmap]`.